Source/WebCore/css/typedom/CSSStyleValueFactory.cpp

```diff
@@ -281,15 +281,15 @@ ExceptionOr<Ref<CSSStyleValue>> CSSStyleValueFactory::reifyValue(const CSSValue&
         default:
             break;
         }
-    } else if (is<CSSImageValue>(cssValue))
-        return Ref<CSSStyleValue> { CSSStyleImageValue::create(downcast<CSSImageValue>(const_cast<CSSValue&>(cssValue)), document) };
-    else if (is<CSSVariableReferenceValue>(cssValue)) {
-        return Ref<CSSStyleValue> { CSSUnparsedValue::create(downcast<CSSVariableReferenceValue>(cssValue).data().tokenRange()) };
-    } else if (is<CSSPendingSubstitutionValue>(cssValue)) {
-        return Ref<CSSStyleValue> { CSSUnparsedValue::create(downcast<CSSPendingSubstitutionValue>(cssValue).shorthandValue().data().tokenRange()) };
-    } else if (is<CSSCustomPropertyValue>(cssValue)) {
+    } else if (auto* imageValue = dynamicDowncast<CSSImageValue>(cssValue))
+        return Ref<CSSStyleValue> { CSSStyleImageValue::create(const_cast<CSSImageValue&>(*imageValue), document) };
+    else if (auto* referenceValue = dynamicDowncast<CSSVariableReferenceValue>(cssValue)) {
```

Comment:
Code is inconsistent about whether it has braces. Would be nicer if it was consistent.

Reply:
Ok, I didn't pay particular attention to this initially. I'll take a look to make it more consistent.